Commands/InteractionCommands/MuteInteractions.cs

```diff
@@ -89,5 +89,60 @@ internal class MuteInteractions : ApplicationCommandModule
                     await ctx.RespondAsync($"{Program.cfgjson.Emoji.Error} That user doesn't appear to be muted, *and* an error occurred while attempting to unmute them anyway. Please contact the bot owner, the error has been logged.");
                 }
         }
+
+        [SlashCommand("tqsmute", "Temporarily mute a user in tech support channels.")]
+        [SlashRequireHomeserverPerm(ServerPermLevel.TechnicalQueriesSlayer)]
+        public async Task TqsMuteSlashCommand(
+            InteractionContext ctx,
+            [Option("user", "The user to mute.")] DiscordUser targetUser,
+            [Option("reason", "The reason for the mute.")] string reason = "No reason specified.")
```

Comment:
This should be required for a slash command, it encourages setting a good reason and makes the UX better when entering name and reason.
```suggestion
            [Option("reason", "The reason for the mute.")] string reason)
```